contracts/DNSRegistrar.sol

```diff
@@ -13,6 +13,13 @@ contract DNSRegistrar {
     DNSSEC public oracle;
     ENS public ens;
 
+    bytes4 constant private INTERFACE_META_ID = bytes4(keccak256("supportsInterface(bytes4)"));
+    bytes4 constant private DNSSEC_CLAIM_ID = bytes4(
```

Comment:
same thing here, why are we combining these?

Reply:
It's an EIP 165 interface ID, which consists of the XOR of all the individual function IDs in the interface.